kalite/coachreports/features/steps/coachreports.py

```diff
@@ -249,7 +249,8 @@ def impl(context):
 
 @when(u"I click on the partial colored cell")
 def impl(context):
-    click_and_wait_for_id_to_appear(context, find_css_with_wait(context, "td.partial"), "details-panel-view")
+    click_and_wait_for_id_to_appear(context, find_css_with_wait(context, "td.partial"), "details-panel-view",
+                                    wait_time=30)
```

Comment:
No, there'a also this: https://github.com/learningequality/ka-lite/pull/4316/files#diff-bde37810880ca8c8ffa2df6c7ca804d5R89